site2/docs/io-redis-sink.md

```diff
@@ -42,11 +42,11 @@ Before using the Redis sink connector, you need to create a configuration file t
     ```json
     {
         "redisHosts": "localhost:6379",
-        "redisPassword": "fake@123",
-        "redisDatabase": "1",
+        "redisPassword": "mypassword",
+        "redisDatabase": "0",
         "clientMode": "Standalone",
         "operationTimeout": "2000",
-        "batchSize": "100",
+        "batchSize": "1",
```

Comment:
where to put the configuration file? we should tell this to users?

Reply:
The configuration has been explained in above `Property` section, so users can get this.

Reply:
the configuration doc does not tell users where to put the configuration file

https://pulsar.apache.org/docs/en/next/io-redis-sink/#configuration

Reply:
![image](https://user-images.githubusercontent.com/31875453/124532695-a194f100-de43-11eb-865c-f94abfc14488.png)

Reply:
BTW, you can refer to `io-elasticsearch-sink.md` for an example.

Reply:
I mean where to put the .yaml file?

Reply:
Sorry to misunderstand this, the current path you started the Pulsar service  (i.e. usually `$PULSAR_HOME` such as `/apps/apache-pulsar-2.8.0`)

Reply:
thanks for your explanation. we should tell this to users? for example, add this info to the `configuration` section?
![image](https://user-images.githubusercontent.com/50226895/124683542-d49cba80-deff-11eb-97c9-1ab3a5df0b2e.png)

Reply:
Done, PTAL.